frontend/js/marc-record-merge-postmerge-service.js

```diff
@@ -447,6 +462,26 @@ export function removeExtra035aFromMerged(preferredRecord, otherRecord, mergedRe
   };
 }
 
+export function add080VersionCode (preferredRecord, otherRecord, mergedRecordParam) {
+  const mergedRecord = new MarcRecord(mergedRecordParam);
+
+  mergedRecord.fields.filter(field => field.tag === '080')
+    .forEach(field => {
+      if (field.subfields.some(({code}) => code === '2')) {
```

Comment:
Actually, it's better to do the filtering in the `filter` clause above. Change into:
```js
mergedRecord.fields.filter({tag, subfields}) => {
  return tag === '080' &&  has2() === false;

  function has2() {
    return subfields.some(({code}) => code === '2');
  }
});
```